### PR TITLE
LIXA Walls

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
None

#### Purpose of change
Adds special walls to LIXA.  It goes with the location.

#### Describe the solution
Adjusts the transforms on bash. 

#### Describe alternatives you've considered
Low grade, bottom barrel chicanery options

#### Testing
Tested with Pickaxe.  Walls performed appropriately.

### CODE REVIEW

File: data/json/furniture_and_terrain/LIXA_furniture_and_terrain.json [hunk 1]
```diff
@@ -23,6 +23,70 @@
...
+      "str_max": 460,
+      "sound": "scrrrash!",
+      "sound_fail": "whump!",
+      "ter_set": "t_folded_wall"
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_folded_wall",
+    "name": "folded  wall",
```
Comment (Uwuewsky) [inline data/json/furniture_and_terrain/LIXA_furniture_and_terrain.json:51]:
```suggestion
    "name": "folded wall",
```

File: data/json/furniture_and_terrain/LIXA_furniture_and_terrain.json [hunk 2]
```diff
@@ -23,6 +23,70 @@
...
+      "str_max": 460,
+      "sound": "scrrrash!",
+      "sound_fail": "whump!",
+      "ter_set": "t_unfolded_wall"
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_unfolded_wall",
+    "name": "unfolded  wall",
```
Comment (Uwuewsky) [inline data/json/furniture_and_terrain/LIXA_furniture_and_terrain.json:72]:
```suggestion
    "name": "unfolded wall",
```

**Comment by ShnitzelX2** [comment]
t_folded_wall and t_unfolded_wall should probably be indestructible to avoid players mining/bombing into or out of the unfolded space, just my two cents.

**Comment by SirPendrak** [comment]
Lixa Balls

Forgive me, I could not resist

**Comment by Maleclypse** [comment]
> t_folded_wall and t_unfolded_wall should probably be indestructible to avoid players mining/bombing into or out of the unfolded space, just my two cents.

When testing I used a pickaxe and activated it from inventory. Isn’t that mining?

**Comment by ShnitzelX2** [comment]
Sorry, I assumed you were adding unique walls to the unfolded space, and the walls there are indestructible anyways. You're making the _normal_ walls functionally indestructible, which is a good change.

**Pull request merged**